app/screens/login/sso_options.tsx

```diff
@@ -46,8 +46,8 @@ const SsoOptions = ({goToSso, ssoOnly, ssoOptions, theme}: Props) => {
                 sso.imageSrc = require('@assets/images/Icon_Google.png');
                 break;
             case Sso.OFFICE365:
-                sso.text = formatMessage({id: 'mobile.login_options.office365', defaultMessage: 'Office 365'});
-                sso.imageSrc = require('@assets/images/Icon_Office.png');
```

Comment:
File should be deleted

Reply:
thanks for confirming this, I was not sure and forgot to ask 👍